informer/informer_integration_test.go

```diff
@@ -1,233 +0,0 @@
-// +build integration
-
-package informer
-
-/*
-	Usage:
-
-		go test -tags=integration ./informer [FLAGS]
-
-	Flags:
-
-		-integration.address string
-			Kubernetes API server address (default "https://$(minikube ip):8443")
-		-integration.ca string
-			CA file path (default "$HOME/.minikube/ca.crt")
-		-integration.crt string
-			certificate file path (default "$HOME/.minikube/apiserver.crt")
-		-integration.key string
-			key file path (default "$HOME/.minikube/apiserver.key")
-*/
```

Comment:
So far we executed against a local minikube. This is now in CI and I did not find a cool and fast way to keep support for both. Ideas welcome.

Reply:
I think it's possible to run that with e2e-tests binary. @fgimenez ?

Reply:
In fact e2e-harness allows you to run on a local minikube too without changing anything else, having e2e-harness binary just `minikube delete && minikube start --extra-config=apiserver.Authorization.Mode=RBAC && e2e-harness setup --remote=false && e2e-harness test`. This is why it tries to build the project binary and the image, and also why the tests are run inside a container, it works both locally and in CI seamlessly.

Reply:
Nice.